lib/Mail.php

```diff
@@ -101,7 +107,9 @@ public function sendGuestInviteMail(Share\IShare $share, $uid, $token) {
 		$this->logger->debug("sending invite to $shareWith: $registerLink", ['app' => 'guests']);
 
 		$filename = \trim($share->getTarget(), '/');
-		$subject = (string)$this->l10n->t('%s shared »%s« with you', [$senderDisplayName, $filename]);
+		$useDefaultLanguage = $this->getDefaultLanguage();
```

Comment:
```suggestion
		$defaultLanguage = $this->getDefaultLanguage();
```
And adjust the var name in the code.